### DISCUSSION TITLE
Mobile Gestures Possible?

### DISCUSSION THREAD

**Discussion by s3m3nT3s** [easepick/easepick#110] [Ideas]
Has anyone been able to adapt easepick with mobile gestures?
If so, I would be insterested to learn/help achieve this.

Examples:
- Swipe left/right would change to next/previous months; even possible with scroll animation?
- Click and drag to select ranges?
- ...

Best Regrads to all
Great datepicker !!!

**Comment by alex-reach** [comment]
@s3m3nT3s Yes, great feature request!
